src/nimblepkg/options.nim

```diff
@@ -89,8 +89,8 @@ Commands:
   uninstall    [pkgname, ...]     Uninstalls a list of packages.
                [-i, --inclDeps]   Uninstall package and dependent package(s).
   build        [opts, ...] [bin]  Builds a package.
-  run          [opts, ...] bin    Builds and runs a package.
-                                  A binary name needs
+  run          [opts, ...] [bin]  Builds and runs a package.
+                                  If several binaries defined then binary name needs
```

Comment:
I think this english is a little bit wrong, IMO "If there are several binaries defined, then the binary name " would be a bit better

Reply:
updated, Was trying to not make it too long